app/src/main/java/com/example/travelapp/BinaryMinHeap.java

```diff
@@ -111,13 +111,7 @@ public void decreaseKey(V value, Key newKey) {
         int parent = calculateParentIndex.apply(index);
         // call helper function to place newKey in the right place
         int finIndex = placeElementAtIndex(index, parent);
-        int index = indices.get(value);
 
-        // update arraylist
-        a.set(index, new Entry<Key, V>(newKey, value));
-        int parent = (index + 1) / 2 - 1;
-        // call helper function to place newKey in the right place
-        int finIndex = helperPlace(index, parent);
```

Comment:
oh interesting, why are you removing this part?

Reply:
This was part of an older version of the file, but I had renamed the method helperPlace to do the same thing in the couple lines of code right above this. I think I accidentally restored these lines, so I just deleted them since redeclaring the variables index, parent, and finIndex caused issues and isn't necessary.